### PR TITLE
Add geo attribute

### PR DESCRIPTION
**Pull request by thenav56**
* Add geo validation on fieldsight fetch
* Add geo fixtures `python3 manage.py loaddata province.json district.json palika.json`
* Re-create migrations

### CODE REVIEW

File: server/apps/fieldsight/loader.py [hunk 1]
```diff
@@ -156,13 +161,14 @@ def load_geosite(self, datum):
...
-        defaults['district'], _ = District.objects.get_or_create(
-            name=get_attr(datum, 'District')
-        )
-        defaults['gaupalika'], _ = Gaupalika.objects.get_or_create(
-            name=get_attr(datum, 'Gaupalika'),
-            defaults={'district': defaults['district']}
-        )
+        defaults['district'] = District.objects.filter(
+            name__trigram_similar=get_attr(datum, 'District')
+        ).first()
```
Comment (tnagorra) [inline server/apps/fieldsight/loader.py]: We should have a threshold for trigram similarity.

Should log errors for which similarity is below threshold

File: server/apps/fieldsight/loader.py [hunk 2]
```diff
@@ -188,15 +200,16 @@ def load_household(self, datum):
...
 
         try:
-            defaults['district'], _ = District.objects.get_or_create(
-                name=get_attr(datum, 'District_of_origin')
-            )
-            defaults['gaupalika'], _ = Gaupalika.objects.get_or_create(
-                name=get_attr(datum, 'Gaupalika_Municipality'),
-                defaults={'district': defaults['district']}
-            )
+            defaults['district'] = District.objects.filter(
```
Comment (tnagorra) [inline server/apps/fieldsight/loader.py]: We should have a threshold for trigram similarity.

Should log errors for which similarity is below threshold

File: server/apps/fieldsight/loader.py (hunk 1 above)
Comment (tnagorra) [reply server/apps/fieldsight/loader.py]: Created an issue for this at https://github.com/toggle-corp/pin-dashboard/issues/33

**Comment by tnagorra** [comment]
We should change api `http://localhost:8050/api/v1/metadata/` to `http://localhost:8050/api/v1/metadata/country`

We should change api `http://localhost:8050/api/v1/metadata/Gorkha` to `http://localhost:8050/api/v1/metadata/district/<gorkha-id>`

We should also create api for palika level

The variables should be camelCase. (Start with lowercase character and without underscore)
```jsom
    "geohazardAffected": {
        "Eligible": 526,
        "Relocated": 329,
        "Total": 526
    },
    "cat2_points": {},
    "cat3_points": {},
```

If there is no data, we should return null/undefined instead of zero.

**Comment by tnagorra** [comment]
Also, the centroid and bbox data has been updated here:
https://drive.google.com/drive/folders/1JC-zm1ygeXcnsS0h4lHXd0yp-z9pOb4V

We will be needing the bbox and centroid data for a particular region.
Will supply with the bbox and centroid for Nepal later, which is not available in the drive.

File: server/apps/fieldsight/loader.py (hunk 1 above)
Comment (thenav56) [reply server/apps/fieldsight/loader.py]: Done

File: server/apps/fieldsight/loader.py (hunk 2 above)
Comment (thenav56) [reply server/apps/fieldsight/loader.py]: Done

**Comment by thenav56** [comment]
@tnagorra Pushed with feedback changes.

**Pull request merged**